cmd/csi-cluster-driver-registrar/main.go

```diff
@@ -141,3 +145,24 @@ func buildConfig(kubeconfig string) (*rest.Config, error) {
 	// kubernetes.
 	return rest.InClusterConfig()
 }
+
+func isAttachRequired(ctx context.Context, conn *grpc.ClientConn) (bool, error) {
+	client := csi.NewControllerClient(conn)
+
+	req := csi.ControllerGetCapabilitiesRequest{}
+
+	rsp, err := client.ControllerGetCapabilities(ctx, &req)
```

Comment:
Can you use [GetControllerCapabilities](https://github.com/kubernetes-csi/csi-lib-utils/blob/master/rpc/common.go#L84) instead?

Reply:
good catch, done!